internal/controllers/machine/drain/drain.go

```diff
@@ -301,12 +301,12 @@ func (r EvictionResult) DrainCompleted() bool {
 }
 
 // ConditionMessage returns a condition message for the case where a drain is not completed.
-func (r EvictionResult) ConditionMessage() string {
+func (r EvictionResult) ConditionMessage(nodeDrainStartTime *metav1.Time) string {
 	if r.DrainCompleted() {
 		return ""
 	}
 
-	conditionMessage := "Drain not completed yet:"
+	conditionMessage := fmt.Sprintf("Drain not completed yet (started at %s):", nodeDrainStartTime.Format(time.RFC3339))
```

Comment:
I think I am doing the same for volume detach

```go
s.deletingMessage = fmt.Sprintf("Waiting for Node volumes to be detached (started at %s)",
```

Reply:
Doesn't this break grouping during aggregation? different machines have a different timestamp?

But we may not want to aggregate this message anyway because it usually is too verbose, right?

Reply:
@chrischdi Are we aggregating Deleting conditions by message?

Reply:
Aren't we getting around this by bucketing stuff in less than 30m / more than 30m?

Reply:
This is the Deleting condition. Not the message we surface in the Ready condition

Reply:
Sorry, I wasn't clear.
IMO It is ok to have a detailed Deleting condition message, because we are never going to use it for aggreagation.
For aggregation we use a simplified message bucketing machines in two categories (less than 30m / more than 30m)

Reply:
To give an example.
```
Drain not completed yet (started at 2024-10-09T16:13:59Z):
* Pods with deletionTimestamp that still exist: pod-2-deletionTimestamp-set-1, pod-2-deletionTimestamp-set-2, pod-2-deletionTimestamp-set-3, pod-3-to-trigger-eviction-successfully-1, pod-3-to-trigger-eviction-successfully-2, ... (2 more)
* Pods with eviction failed:
  * Cannot evict pod as it would violate the pod's disruption budget. The disruption budget pod-5-pdb needs 20 healthy pods and has 20 currently: pod-5-to-trigger-eviction-pdb-violated-1, pod-5-to-trigger-eviction-pdb-violated-2, pod-5-to-trigger-eviction-pdb-violated-3, ... (3 more)
  * some other error 1: pod-6-to-trigger-eviction-some-other-error
  * some other error 2: pod-7-to-trigger-eviction-some-other-error
  * some other error 3: pod-8-to-trigger-eviction-some-other-error
  * some other error 4: pod-9-to-trigger-eviction-some-other-error
  * ... (1 more error applying to 1 Pod)`
```

It's very very unlikely that this will every look the same for different Machines, even if I get rid of the start time entirely)